packages/cli/src/tasks/eject.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = ejectConfigFiles = async (copyAllFiles) => {
+  return new Promise(async (resolve, reject) => {
+    try {
+
+      if (copyAllFiles) {
+        configFilePaths = fs.readdirSync(path.join(__dirname, '../config'));
+      } else {
+        configFilePaths = [
```

Comment:
I know by default **webpack** will automatically pickup a _webpack.config.js_ file.  Do we think it would make sense to follow that convention?  In regards to #370, might make it easier picking the files up on the other end.

Reply:
We need this path for our tasks.